kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -356,11 +359,11 @@ The `#next [_]` operator initiates execution by:
 -   `#stackDelta` is the delta the stack will have after the opcode executes.
 
 ```k
-    syntax Bool ::= #stackUnderflow ( WordStack , OpCode ) [symbol(#stackUnderflow), macro]
-                  | #stackOverflow  ( WordStack , OpCode ) [symbol(#stackOverflow), macro]
+    syntax Bool ::= #stackUnderflow ( Int , OpCode ) [symbol(#stackUnderflow), macro]
+                  | #stackOverflow  ( Int , OpCode ) [symbol(#stackOverflow), macro]
  // ---------------------------------------------------------------------------------------
-    rule #stackUnderflow(WS, OP:OpCode) => #sizeWordStack(WS) <Int #stackNeeded(OP)
-    rule #stackOverflow (WS, OP) => #sizeWordStack(WS) +Int #stackDelta(OP) >Int 1024
+    rule #stackUnderflow(WSSize, OP:OpCode) => WSSize <Int #stackNeeded(OP)
```

Comment:
Could you change this to all caps for consistency?
```suggestion
    rule #stackUnderflow(WSSIZE, OP:OpCode) => WSSize <Int #stackNeeded(OP)
```

Reply:
Yes, good idea, I have done this.